tests/resources/test-config-template.yaml

```diff
@@ -11,7 +11,7 @@ security_server:
     - /path/to/authcert
   configuration_anchor: /path/to/configuration-anchor.xml
   name: <SECURITY_SERVER_NAME>
-  owner_dn_country: FI
+  owner_dn_country: EE
```

Comment:
Please revert this change, because in order for these tests to successfully run on Jenkins, this owner_dn_country needs to be "FI".